path-manager/conf/routes

```diff
@@ -5,7 +5,7 @@ GET     /paths                              controllers.PathManagerController.ge
 
 POST    /paths                              controllers.PathManagerController.registerNewPath
 PUT     /paths/:id                          controllers.PathManagerController.registerExistingPath(id: Long)
-POST    /paths/:id                          controllers.PathManagerController.updateCanonicalPath(id: Long)
+POST    /paths/:id                          controllers.PathManagerController.updateCanonicalPath(id: Long, formAlias: Boolean)
```

Comment:
see my comment https://github.com/guardian/path-manager/pull/33#discussion_r467952889

Reply:
also i think the PR description needs updating as it refers to `fromAlias`